content/en/docs/tasks/run-application/access-api-from-pod.md

```diff
@@ -76,7 +76,7 @@ directly to the API server.  The internal certificate secures the connection.
 
 ```shell
 # Point to the internal API server hostname
-APISERVER=https://kubernetes.default.svc
+APISERVER=https://kubernetes.default.svc.cluster.local
```

Comment:
Here's the downside of this change: we don't know what this cluster's cluster domain is (and the [downward API](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/) doesn't tell us).

Reply:
Hi @sftim thanks for looking at this.  I don't know fully why this doesn't work on any cluster available to me.  My primary clusters are all EKS so resolve.conf is almost certainly a subject of their EKS images.

The other clusters I have available is Kind.  And if you do a `kind create cluster`, this doesn't work either. But I couldn't find what is the default resolve.conf injected into pods by kubelet (is that even relevant here?). In other words, does this just not work on EKS and Kind, and these are the outliers. If so, then the docs should probably stay as they are.